dist/index.html

```diff
@@ -2,12 +2,290 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Title</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500&display=swap&subset=cyrillic,cyrillic-ext">
+    <link rel="stylesheet" href="https://meyerweb.com/eric/tools/css/reset/reset.css">
     <link rel="stylesheet" href="./styles/main.css">
   </head>
   <body>
-    <img src="./images/logo.svg" alt="Logo">
-    <h1>Hello Mate academy</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
+  <header>
+    <div class="header-menu">
+      <div class="header-block">
+        <h1 class="uber-logo">
+          <a href="#">
+            <img src="images/logo-header.png" alt="ubereats logo"  class="header-logo">
+          </a>
+        </h1>
+        <div class="when-and-where">
+          <form>
+            <label class="when">
+              <input
+                class="on-time"
+                placeholder="ASAP"
+                type="text"
+                name="when">
+            </label>
+          </form>
+          <p class="to text-light">to</p>
+          <form class="where">
+            <label class="where">
+              <img src="images/place.png" alt="place">
+              <input
+                placeholder="ul. Tarasivska St, 16"
+                type="text"
+                name="where">
+            </label>
+          </form>
+        </div>
+        <div class="buttons-and-basket">
+          <button class="sign-in text-medium">
+            <a href="#">Sign In</a>
+          </button>
+          <button class="registration text-medium">
+            <a href="#">Register</a>
+          </button>
+          <h2>
+            <a href="#">
+              <img src="images/basket.png" alt="basket"  class="basket">
+            </a>
+          </h2>
+        </div>
+      </div>
+    </div>
+    <form class="search-form ">
+      <label class="search-form__label">
+        <img src="images/Search.png" alt="search" class="search-form_logo">
+        <input
+          class="search-form__input text-normal"
+          type="text"
+          placeholder="Search for restaurant or cusine">
+      </label>
+    </form>
+    <div class="mobile-when-and-where">
+      <form>
+        <label
+          class="label text-normal"
+          for="when">When</label>
+        <label class="when">
```

Comment:
why do you need another `<label>`? this is not a real label, choose another tag, e.g. `<div>`